angular/core/models/membership_model.coffee

```diff
@@ -20,9 +20,24 @@ angular.module('loomioApp').factory 'MembershipModel', (BaseModel, AppConfig) ->
     groupName: ->
       @group().name
 
-    changeVolume: (volume) ->
-      @volume = volume
-      @save()
+    volume: ->
+      @volumeValue
+
+    saveVolume: (volume, applyToAll = false, setDefault = false) ->
+      @volumeValue = volume
+      @applyToAll = applyToAll
+      @setDefault = setDefault
+      @save().then =>
```

Comment:
I'm not too sure about having this things as attributes on the model... I know what a discussion's title is, or a group's privacy, but I'm not sure what a membership's 'applyToAll' is... these are details about the particular post, and don't really have to do with the model itself.

Which makes me think that `save()` isn't the right way to go here, I think we want to move the out into a separate controller endpoint:

```
@remote.postMember 'update_volume', @id,
  volume: volume
  apply_to_all: applyToAll
  set_default: setDefault
```

```
# api/memberships_controller
def change_volume
  service.change_volume(membership: load_resource, params: params, actor: current_user)
  respond_with_resource
end
```

```
# membership_service
def self.change_volume(membership:, params:, actor:)
  actor.ability.authorize! :update, membership
  if (params.slice(:set_default) { update stuff }
  if (params.slice(:apply_to_all) { update stuff }
  EventBus.broadcast('membership_change_volume') { }
end
```